src/Oauth2CredentialManagers/BaseCredentialManager.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Webfox\Xero\Oauth2CredentialManagers;
+
+abstract class BaseCredentialManager
+{
+    public function getAccessToken(): string
+    {
+        return $this->data('token');
```

Comment:
We should probably declare `data` as `protected function data(string $key = null)` on this class.